pkg/webhooks/serve.go

```diff
@@ -0,0 +1,48 @@
+package webhooks
+
+import (
+	"os"
+	"path/filepath"
+
+	logger "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"kubevirt.io/node-maintenance-operator/pkg/apis/nodemaintenance/v1beta1"
+)
+
+const (
+	WebhookPort     = 8443
+	WebhookCertDir  = "/apiserver.local.config/certificates"
+	WebhookCertName = "apiserver.crt"
+	WebhookKeyName  = "apiserver.key"
+)
+
+var log = logger.Log.WithName("webhook")
+
+func Serve(mgr manager.Manager, stop <-chan struct{}) error {
+
+	// Make sure the certificates are mounted, this should be handled by the OLM
```

Comment:
We were looking in the wrong place before?

Reply:
no, when I mentioned yesterday that certs are not mounted yet I missed that the webhook's deploymentName [0] has to match the operator's deploment name in the CSV, and the former is derived from the service name in the webhook template [1]. So all good, it was just work in progress.

[0] https://github.com/kubevirt/node-maintenance-operator/pull/102/files#diff-0e416e752895627bc1a91e19c6be6e3cR200
[1] https://github.com/kubevirt/node-maintenance-operator/pull/102/files#diff-0132d87c0cf6c4433c9fcac6a4a769bdR12